flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyPartitionRequestClient.java

```diff
@@ -277,6 +270,16 @@ private boolean canBeReused() {
         return clientFactory.isConnectionReuseEnabled() && !clientHandler.hasChannelError();
     }
 
+    @VisibleForTesting
+    public Channel getTcpChannel() {
```

Comment:
I think we can get the handler directly through the channel(via `channel.pipeline().get(NetworkClientHandler.class`), so we don't need to introduce the next method.